.storybook/preview.js

```diff
@@ -37,9 +37,9 @@ const ThemeContainer = ({ theme, contentDensity, children, direction }) => {
     }
   }, [contentDensity]);
 
-  useEffect(() => {
-    document.querySelector('html').setAttribute('dir', direction.toLowerCase());
-  }, [direction]);
+  // useEffect(() => {
+  //   document.querySelector('html').setAttribute('dir', direction.toLowerCase());
+  // }, [direction]);
```

Comment:
Was this intentionally not removed?

Reply:
Yes, because we need some way to change the Text Direction in the storybook but this is only half of the solution. I just didn't figure out yet how to change the RTL setting of the Web Componenents